src/pages/profitSharing/browse/copyTradersBrowse.scss

```diff
@@ -0,0 +1,3 @@
+.ctBrowsePage {
```

Comment:
same below but I guess not an issue for now

Reply:
Done